egs/wsj/s5/steps/nnet3/chain/get_egs.sh

```diff
@@ -271,14 +279,16 @@ if [ $stage -le 2 ]; then
   for id in $(seq $num_lat_jobs); do cat $dir/lat.$id.scp; done > $dir/lat.scp
 fi
 
+[ $ivector_interval -gt $frames_per_eg ] && echo "Error ivector_interval should not be greater than frames_per_eg." && exit 1
```

Comment:
it's not clear to me that it would matter if ivector_interval was greater than frame_per_eg.  Also, I think we're going to call it ivector_period, right?